packages/database/src/migrations/20211128090124-MergeFlutrackingResponses-modifies-data.js

```diff
@@ -158,19 +158,20 @@ const mergeFlutrackingResponses = async (db, surveyCode) => {
   );
   console.log(`Batches found: ${responseGroups.length}`);
 
-  let prevPercentage = -1;
-  await processInBatches(responseGroups, RESPONSE_GROUP_BATCH_SIZE, async (responseGroup, i) => {
+  // Note: we don't process all batches because it takes a really long time and may stall
+  // other migrations. The execution strategy is:
+  // 1. Run the migration once with a small subset of data
+  // 2. Manually re-trigger the migration as many times as required until all data are processed
+  const batchesToProcessCount = 1;
+  const groupsToProcessCount = RESPONSE_GROUP_BATCH_SIZE * batchesToProcessCount;
+  const groupsToProcess = responseGroups.slice(0, groupsToProcessCount);
+  console.log(`This run will process ${groupsToProcess.length} batches`);
+
+  await processInBatches(groupsToProcess, RESPONSE_GROUP_BATCH_SIZE, async (groupBatch, i) => {
     await db.runSql('START TRANSACTION');
-
-    // Print progress
-    const percentage = getPercentage(i, responseGroups.length);
-    if (parseInt(percentage - prevPercentage) > 0) {
-      // Print percentage when its integer part changes
-      console.log(`${Math.round(percentage)}%`);
-      prevPercentage = percentage;
-    }
-
-    await processResponseGroups(db, responseGroup);
+    // Print percentage when its integer part changes
```

Comment:
This comment is no longer needed - I assume because it's a smaller total number now, we don't need to stress about the spam of printing every time

Reply:
Indeed! 👍